src/util/string.h

```diff
@@ -21,6 +21,11 @@
     return spanparsing::Split<std::string>(str, sep);
 }
 
+[[nodiscard]] inline std::vector<std::string> SplitString(std::string_view str, std::string_view separators)
```

Comment:
nit: Maybe also fuzz this? Untested diff:


```diff
diff --git a/src/test/fuzz/string.cpp b/src/test/fuzz/string.cpp
index e6064d19b6..94399faf04 100644
--- a/src/test/fuzz/string.cpp
+++ b/src/test/fuzz/string.cpp
@@ -224,7 +224,12 @@ FUZZ_TARGET(string)
         int64_t amount_out;
         (void)ParseFixedPoint(random_string_1, fuzzed_data_provider.ConsumeIntegralInRange<int>(0, 1024), &amount_out);
     }
-    (void)SplitString(random_string_1, fuzzed_data_provider.ConsumeIntegral<char>());
+    {
+        const auto single_split{SplitString(random_string_1, fuzzed_data_provider.ConsumeIntegral<char>())};
+        assert(single_split.size() >= 1);
+        const auto any_split{SplitString(random_string_1, random_string_2)};
+        assert(any_split.size() >= 1);
+    }
     {
         (void)Untranslated(random_string_1);
         const bilingual_str bs1{random_string_1, random_string_2};

Reply:
I have no experience with fuzzing, would it make sense to add stronger asserts? E.g. check that the sum of all split string length plus number of splits minus 1 should give exactly the input string length:
```cpp
const auto single_split{SplitString(random_string_1, fuzzed_data_provider.ConsumeIntegral<char>())};
size_t sum = 0;
for (auto const& split : single_split) {
    sum += split.size();
}
assert(sum + single_split.size() - 1 == random_string_1.size());

const auto any_split{SplitString(random_string_1, random_string_2)};
sum = 0;
for (auto const& split : any_split) {
    sum += split.size();
}
assert(sum + any_split.size() - 1 == random_string_1.size());
```

Reply:
Sure, just like with other tests, there is no limit to how much can be tested